conjure-java-core/src/integrationInput/java/com/palantir/product/UndertowErrorService.java

```diff
@@ -0,0 +1,31 @@
+package com.palantir.product;
+
+import com.palantir.tokens.auth.AuthHeader;
+import javax.annotation.processing.Generated;
+
+@Generated("com.palantir.conjure.java.services.UndertowServiceInterfaceGenerator")
+public interface UndertowErrorService {
+    /**
+     * @apiNote {@code GET /base/basic}
+     * @throws InvalidArgument
+     */
+    String testBasicError(AuthHeader authHeader) throws ServerTestErrors.InvalidArgument;
+
+    /**
+     * @apiNote {@code GET /base/imported}
+     * @throws EndpointError
+     */
+    String testImportedError(AuthHeader authHeader) throws ServerEndpointSpecificErrors.EndpointError;
+
+    /**
+     * @apiNote {@code GET /base/multiple}
+     * @throws InvalidArgument
+     * @throws NotFound Something was not found.
+     * @throws DifferentNamespace
```

Comment:
I should add the package and namespace here.

Reply:
I think we want a javapoet `$T` reference using the `TypeName` of the error, that way javadoc generation links to the exception